http-client/src/main/java/io/airlift/http/client/HttpClientConfig.java

```diff
@@ -218,6 +220,34 @@ public HttpClientConfig setMaxContentLength(DataSize maxContentLength)
         return this;
     }
 
+    @NotNull
+    public DataSize getRequestBufferSize()
+    {
+        return requestBufferSize;
+    }
+
+    @Config("http-client.request-buffer-size")
+    @MaxDataSize("32MB")
```

Comment:
Bean validations need to go on the getter (the validation framework validates the entire object after it is built)